backend/lib/til/accounts/user.ex

```diff
@@ -14,8 +15,12 @@ defmodule Til.Accounts.User do
   end
 
   def changeset(struct, params) do
+    params = if params["password"],
```

Comment:
it would be better to extract that to simple hash_password or sth function and pipe it then.